.github/workflows/timeseries-scans-virus-bandit.yaml

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2025 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
-name: "[Time SeriesAnalytics] Virus and Bandit Scan"
-run-name: "[Time Series Analytics] Virus and Bandit Scan workflow (by @${{ github.actor }} via ${{ github.event_name }})"
+name: "[Time SeriesAnalytics] Virus, Bandit and Pylint Scan"

```

Comment:
Let's update the filename to add pylint to the list.

It's good to have pylint in the PR workflow as part of the exploration on to reuse these workflow steps there.